Modelica/Mechanics/MultiBody/Frames/planarRotation.mo

```diff
@@ -21,7 +21,7 @@ R = Frames.<strong>planarRotation</strong>(e, angle, der_angle);
 <h4>Description</h4>
 <p>
 The function call <code>Frames.<strong>planarRotation</strong>(e, angle, der_angle)</code> returns
-orientation object R that describes the orientation to rotate in the plane along unit
+orientation object R that describes the orientation of planar rotation along unit
```

Comment:
```suggestion
orientation object R that describes the orientation of a planar rotation about unit
```
I think English normally has an article ("a" or "the") in these cases; and "along" does for me give the wrong impression - https://en.wikipedia.org/wiki/Rotation_matrix#In_three_dimensions uses "about".
(Writing "orthogonal to" seems excessive.)

Reply:
Or did I completely misunderstand the mindset?

Reply:
No, you got it perfectly!